src/commonMain/kotlin/data/gems/RelentlessEarthstormDiamond.kt

```diff
@@ -24,5 +25,13 @@ class RelentlessEarthstormDiamond : Gem(32409, "Relentless Earthstorm Diamond",
         }
     }
 
+    override fun metaActive(sockets: List<Socket>): Boolean {
```

Comment:
This is currently the default implementation for `metaActive` in the `Gem` superclass, shared with Destructive Skyfire Diamond.  

I'd be OK with either leaving this un-overridden in RED, or changing the `Gem` `metaActive` method to `abstract` and moving this implementation to both places, but the override here as written is redundant.

Reply:
Good to know, I had to dive a bit into that space and didn't immediately notice. I'll just remove this to keep it simple.